frontend/packages/console-shared/locales/en/console-shared.json

```diff
@@ -182,7 +182,7 @@
   "Run in Web Terminal": "Run in Web Terminal",
   "Successfully copied to clipboard!": "Successfully copied to clipboard!",
   "Running in Web Terminal": "Running in Web Terminal",
-  "Console plugin enablement - {{plugin}}": "Console plugin enablement - {{plugin}}",
+  "Console plugin enablement - {{pluginName}}": "Console plugin enablement - {{pluginName}}",
```

Comment:
Hmm, I guess we don't have to retranslate since you've edit the messages manually. This should be OK.

Reply:
updated